samcli/hook_packages/terraform/hooks/prepare.py

```diff
@@ -688,6 +709,53 @@ def _get_relevant_cfn_resource(
     )
 
 
+def _get_python_command_name() -> str:
+    """
+    Verify that python is installed and return the name of the python command
+
+    Returns
+    -------
+    str
+        The name of the python command installed
+    """
+    command_names_to_try = ["python", "python3"]
```

Comment:
Will these work in Windows?
Also, is it possible for `python` be pointed to Python2? Is the script being run Python version agnostic?

Reply:
Good points.

At least one of these options should work on Windows. I believe the default one on windows might be `python`, but I can add `py` and `py3` to `command_names_to_try` just in case

The script which is the intended use case for this function should work with python 2 or 3, but @sriram-mv  can confirm. Here's the PR: https://github.com/aws/aws-sam-cli/pull/4117
Regardless, this is an independent function that could be used for other scripts, so I can at least add some comments here to point that out.

Reply:
The script should work across all python versions.